core/controllers/editor_test.py

```diff
@@ -3080,6 +3134,13 @@ def test_return_error_when_image_not_uploaded(self):
             feconf.EXPLORATION_IMAGE_UPLOAD_PREFIX,
             feconf.ENTITY_TYPE_EXPLORATION, exp_id)
 
+        # Check if the file is already present or not.
```

Comment:
Be more definite: "Check that the file is not already present."

Ditto in 3 places below (note that the last one is different).

Reply:
updated at all the locations.